src/hotspot/share/opto/gcm.cpp

```diff
@@ -1130,11 +1130,42 @@ void PhaseCFG::latency_from_uses(Node *n) {
   set_latency_for_node(n, latency);
 }
 
+//------------------------------is_cheaper_block-------------------------
+// Check if a block between early and LCA block of uses is cheaper by
+// frequency-based policy, latency-based policy and random-based policy
+bool PhaseCFG::is_cheaper_block(Block* LCA, Node* self, uint target_latency,
+                                uint end_latency, double least_freq,
+                                int cand_cnt, bool in_latency) {
+  if (StressGCM && C->randomized_select(cand_cnt)) {
+    // Should be randomly accepted in stress mode
+    return true;
+  }
+
+  if (!StressGCM) {
```

Comment:
It could be simplified so you don't need to check `if (!StressGCM)`:
```
  if (StressGCM) {
    // Should be randomly accepted in stress mode
    return C->randomized_select(cand_cnt);
  }
```